styles.css

```diff
@@ -124,12 +137,18 @@ a:hover {
 }
 
 .finished {
-  background: #065006;
+  background: #70C1B3;
 }
 
 
-@media only screen and (max-width: 600px) {
+@media only screen and (max-width: 725px) {
   .container {
-    grid-template-columns: 1fr;
+    grid-template-columns: repeat(2, 1fr);
   }
 }
+
+@media only screen and (max-width: 550px) {
+  .container {
+    grid-template-columns: 1fr;
+  }
+}
```

Comment:
Ah, you need to update your editor to add a newline at the end of the file by default. That's what that little red icon means. It's a throwback to an old file incompatibility thing on Unix.